pkg/interactors/create_user.go

```diff
@@ -0,0 +1,19 @@
+package interactors
+
+import "github.com/CESARBR/knot-babeltower/pkg/logging"
+
+// CreateUser to interact to user
+type CreateUser struct {
+	logger logging.Logger
+}
+
+// NewCreateUser contructs the interactor
+func NewCreateUser(logger logging.Logger) *CreateUser {
+	return &CreateUser{logger}
+}
+
+// Execute runs the interactor
```

Comment:
I think "runs the use case" is more descriptive. What do you think?

Reply:
Done